src/ngx_http_lua_worker.c

```diff
@@ -130,4 +133,41 @@ ngx_http_lua_ffi_worker_count(void)
 
     return (int) ccf->worker_processes;
 }
+
+
+int
+ngx_http_lua_ffi_get_process_type(void)
+{
+    int typ = ngx_process;
+
+    if (typ == NGX_PROCESS_HELPER) {
+#ifdef HAVE_PRIVILEGED_PROCESS_PATCH
```

Comment:
Seems like we should enclose the outer `if` statement inside this macro as well.